vector/src/main/java/fr/gouv/tchap/activity/TchapLoginActivity.java

```diff
@@ -614,6 +610,27 @@ public void onBackPressed() {
         }
     }
 
+    private void promptUserBeforeCancellingEmailPolling() {
+        mCurrentDialog = new AlertDialog.Builder(TchapLoginActivity.this)
```

Comment:
If `onRegistrationSuccess()` is called while this dialog is displayed, another dialog may be displayed. It is tested ok?
Maybe if `mCurrentDialog` is not null in `onRegistrationSuccess()`, it should be dismissed before being assigned again with a new dialog.